packages/web-pkg/src/components/sideBar/Details/SpaceNoSelection.vue

```diff
@@ -0,0 +1,19 @@
+<template>
```

Comment:
I think the `Details` folder should have another parent folder named `Spaces`. And then please move `SpaceNoSelection.vue` one level up. Result should be this layout:

```
- Spaces
  - Details
    - SpaceDetails.vue
    - SpaceDetailsMultiple.vue
  - SpaceNoSelection.vue
```

In a followup PR you could make use of the `SpaceNoSelection` component in the `Projects.vue` view of the files app.